docs/usage/updating-rebasing.md

```diff
@@ -29,7 +29,7 @@ It is up to you to either finish the job and merge the PR, or rename it and clos
 <!-- prettier-ignore -->
 !!! warning
     Do _not_ amend Renovate's commits, because Renovate will rebase over your amended commit.
-    Keep your work safe and always push your own _new_ commit to any Renovate branch.
+    Keep your work safe and never push your own _new_ commit to any Renovate branch.
```

Comment:
```suggestion
    Instead, keep your work safe by pushing _new_ commit on top of the Renovate branch, with a different identity to Renovate's.
```

@pfuhrmann do you understand the meaning of this sentence now, with my proposed rewording?

Reply:
Do I understand it correctly what you mean here is actually `git amend`?

Maybe the first sentence could read "Do _not_ `git amend` Renovate's commits because Renovate will rebase over your amended commit.".

I would argue pushing any commits to any Renovate branch is "dangerous". PRs can be recreated by a simple checkbox in the dashboard which will cause the loss of changes too. Therefore personal commits are not safe in any Renovate PR hence my earlier suggestion of "never" pushing to Renovate branches.

Reply:
I understand now however what you mean will close this PR and let you decide on the wording :+1: